recipes-domd/agl/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7796-salvator-x-domd.dts

```diff
@@ -1021,3 +997,4 @@
 &dmac2 {
 	/delete-property/iommus;
 };
+
```

Comment:
It is preferred to do whitespace changes in separate patch.

Reply:
done